act-rules-format.bs

```diff
@@ -775,37 +773,28 @@ Examples in this section include:
   "hasPart": "dct:hasPart",
   "source": "dct:source",
 
-  "Agent": "foaf:term_Agent",
```

Comment:
None of these need to be prefaced with `term_`. Have a look at this FOAF example. It shows the correct namespace and term to use: http://xmlns.com/foaf/spec/#sec-example